### PR TITLE
Style cleanup for table-bordered and warning-text

### PR DESCRIPTION
**Pull request by mkayan**

### CODE REVIEW

File: app/kitchensink/index.html [hunk 1]
```diff
@@ -2430,233 +2430,59 @@ <h2 id="Tables" class="page-header">Tables
...
-                            <span>Improve account management workflow</span>
-                          </td>
-                          <td>
-                            <span>$6,000</span>
-                          </td>
-                        </tr>
-                      </tbody>
-                    </table> -->
+                  <h4>Includes Borderless and Hover stylings with MDL additions <small>Just by using the classnames: "table mdl-js-data-table"</small></h4>                  
+                   <!--  <table class="table mdl-js-data-table">
```
Comment (davezen1) [inline app/kitchensink/index.html]: Why are we removing a table example?

File: app/styles/custom/_variables.scss [hunk 2]
```diff
@@ -35,7 +35,7 @@ $brand-default-border:              #adadad;
 $brand-live-border:                 darken($brand-live, 10%);
 
 // custom brand text
```
Comment (davezen1) [inline app/styles/custom/_variables.scss]: we need a comment for why this color is here

File: app/styles/custom/_variables.scss [hunk 3]
```diff
@@ -35,7 +35,7 @@ $brand-default-border:              #adadad;
...
-$brand-warning-text:                darken($brand-warning, 45%);
+$brand-warning-text:               #A44900;
```
Comment (davezen1) [inline app/styles/custom/_variables.scss]: we need a comment why this color is here and not a calc #A44900;

File: app/kitchensink/index.html (hunk 1 above)
Comment (mkayan) [reply app/kitchensink/index.html]: It was commented out. I removed the comments and checked in the changes.

**Pull request merged**